istioctl/cmd/proxyconfig.go

```diff
@@ -591,6 +591,11 @@ func logCmd() *cobra.Command {
 				if err != nil {
 					return err
 				}
+				if len(podNames) > 1 {
+					return fmt.Errorf("multiple pods(%v) selected, you should chose one specified pod", podNames)
```

Comment:
Instead of returning an error, you should select the first pod by default, or ask a user to pass pod name as a selector?